src/main/include/subsystems/ActuatorSubsystem.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <frc/Servo.h>
+#include <frc2/command/SubsystemBase.h>
+
+#include "frc/Joystick.h"
+
+class ActuatorSubsystem : public frc2::SubsystemBase
+{
+public:
+    ActuatorSubsystem();
+    void Periodic() override;
+    void SetActutorPoistion();
+
+private:
+    frc::Servo m_left{1};
+    frc::Servo m_right{2};
+    frc::Joystick m_ActuatorJoystick{0};
```

Comment:
maybe a better idea to make the subsystem take in a single servo port and just make two subsystem objects. also keep the joysticks in RobotContainer and pass in a desired position via a default command.